api/envoy/config/core/v3/base.proto

```diff
@@ -303,6 +303,50 @@ message RuntimeFeatureFlag {
   string runtime_key = 2 [(validate.rules).string = {min_len: 1}];
 }
 
+message KeyValue {
+  // The key of the key/value pair.
+  string key = 1 [(validate.rules).string = {min_len: 1}];
+
+  // The value of the key/value pair.
+  string value = 2;
```

Comment:
What about non-UTF8? This has come up a few times with headers.

Reply:
I initially think we can add `raw_value` with type bytes in future if we need it. Should I change it to bytes directly?

Reply:
I think it's generally considered a mistake that we had `string` instead of `bytes` for header values, CC @yanavlasov

Reply:
@wbpcode @yanavlasov bytes or strings? I'm pretty sure we should use bytes if we expect at all the possibiltiy of non-UTF8 here. At the wire level, for proto3, these are identical.